app/models/user.rb

```diff
@@ -3,10 +3,19 @@
 class User < ApplicationRecord
   include Clearance::User
 
+  has_many :activity_notifications,
+           as: :recipient,
+           dependent: :destroy
+  has_many :notifications, class_name: 'ActivityNotification', dependent: :destroy
```

Comment:
maybe we should consider renaming these to make it easier to understand?

like `user.notifications_as_recipient` and `user/message/request/contributor.notifications_as_mentioned` or something similar?

Reply:
I don't know if I like the semantics of `notifications_as_sender`... In the case the `user` has their id saved to the `activity_notifications`  table, there would also be a `contributor_id`, `message_id`, and `request_id` since this only happens for `ChatMessageSent` types. In this case, they would all be the "sender"?